recipe/meta.yaml

```diff
@@ -63,6 +67,8 @@ source:
 
 build:
   number: {{ build }}
+  skip: true  # [py!=311]
```

Comment:
Why are we skipping all the other python versions? I understand for development, but this got merged like that?

Reply:
Ah sorry, saw that @hmaarrfk already addressed this. I was just looking through the diff and noticed how we're deleting almost all CI configs...

Reply:
right, sorry, we were trouble shooting patches, so i didn't want to hog all the CIs during the debugging process.

50 adds up quickly.